src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -363,7 +363,6 @@ well as menu structures (for main menu and popup menus).
          <menu label="_Import Dataset">
             <cmd refid="importDatasetFromFile"/>
             <cmd refid="importDatasetFromURL"/>
-            <cmd refid="importDatasetFromCSV"/>
```

Comment:
Not sure I get the context of this change (assuming we'd want this on the main menu as well).